rust/automerge-c/src/doc/list.rs

```diff
@@ -486,7 +486,7 @@ pub unsafe extern "C" fn AMlistPutStr(
 }
 
 /// \memberof AMdoc
-/// \brief Puts a Lamport timestamp as the value at an index in a list object.
+/// \brief Puts an epoch timestamp as the value at an index in a list object.
```

Comment:
We should probably specify "unix epoch" here.

Reply:
and `in milliseconds` please.

```suggestion
/// \brief Puts number of milliseconds since the unix epoch as a timestamp at an index in a list object.
```